core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -653,6 +653,19 @@ class ZkUtils(val zkClient: ZkClient,
     (dataStr, stat)
   }
 
+  def readData(path: String, returnNoneIfPathNotExists: Boolean): Option[String] = {
```

Comment:
Also, given that we already have a `readData` method, my original suggestion doesn't work. So, it needs to be rethought in a JIRA perhaps.